examples/compiler/01_iet.ipynb

```diff
@@ -0,0 +1,664 @@
+    "\n",
+    "With this we can derive symbolic expressions for the backward derivatives in space directly via the `u.dxl` and `u.dyl` shorthand expressions (the l indicates \"left\" or backward differences) and the shorthand notation `u.dt` provided by `TimeFunction` objects to derive the forward derivative in time.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Data([[[0., 0., 0.],\n",
+       "       [0., 0., 0.],\n",
+       "       [0., 0., 0.]],\n",
+       "\n",
+       "      [[0., 0., 0.],\n",
+       "       [0., 0., 0.],\n",
+       "       [0., 0., 0.]]], dtype=float32)"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from devito import Eq, Grid, TimeFunction, Operator\n",
+    "\n",
+    "grid = Grid(shape=(3, 3))\n",
+    "u = TimeFunction(name='u', grid=grid)\n",
+    "u.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we have just declared a two-dimensional domain with three coordinates at each dimension (_x_ and _y_). Each coordinate of such a discrete space will be holding a real value.\n",
+    "\n",
+    "As we can see, we can always access the values at each coordinate of the domain. At this point, no modifications have been done to it yet. `u.data` give us a quick access over the values holded by each cell of such a domain. `u.data[0]` holds the values in the grid at the \"current\" iteration time, given a time-step to be considered, whereas `u.data[1]` holds the values of `u` for the \"current+1\" time-step.\n",
+    "\n",
+    "We can now create an `Operator` that will perform modifications onto our domain according to differential equations through a computational stencil. \n",
+    "It means that those differential equations will be translated into finite differences that will be used to update the values at each coordinate of the space.\n",
+    "Such finite differences, or _expressions_, will be applied for specific ranges of _iterations_ over the domain. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Eq(u(t + dt, x, y), u(t, x, y) + 1)"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "eq = Eq(u.forward, u+1)\n",
+    "op = Operator(eq)\n",
+    "op.args['expressions']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For instance, the particular `Eq` object above allows us say that, at each time step, 1 will be added to every position of the domain.\n",
```

Comment:
This is not what this stencil do. This stencil updates the current time step with the value at the previous time-step + 1.
> at each time step, 1 will be added to every position of the domain.

this would be `Eq(u, u + 1)`